testing/ostinato/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Maintainer: Corentin Henry <corentinhenry@gmail.com>
+pkgname=ostinato
+pkgver=0.7.1
+pkgrel=0
+pkgdesc="Packet/Traffic Generator and Analyzer"
+url="https://www.ostinato.org/"
+arch="all"
+license="GPLv3"
+depends=
+makedepends="qt-dev protobuf-dev libpcap-dev paxctl"
+subpackages="$pkgname-drone $pkgname-gui"
+source="http://dl.bintray.com/pstavirs/ostinato/ostinato-src-0.7.1.tar.gz"
+
+_builddir="$srcdir"/$pkgname-$pkgver
```

Comment:
Rename `_builddir` to `builddir` (this has been changed in abuild tool recently).